configurations/configuration.ts

```diff
@@ -0,0 +1,5 @@
+import nconf = require('nconf');
```

Comment:
this is part of source - you can create folder named `utils` and put there config, logger etc

Reply:
done